support/elasticsearch2_search.erl

```diff
@@ -442,26 +465,37 @@ map_query({text, Text}, Context) ->
         <<"title*^2">>
     ],
     Fields = z_notifier:foldr(#elasticsearch_fields{query = Text}, DefaultFields, Context),
+    DefOpCfg = m_config:get_value(mod_elasticsearch2, default_operator, Context),
+    DefaultOperator = case z_convert:to_upper(z_convert:to_binary(DefOpCfg)) of
+        <<"AND">> = DefOp -> DefOp;
+        <<"OR">> = DefOp -> DefOp;
+        _ -> <<"OR">>
+    end,
     Query = case z_convert:to_bool(m_config:get_value(mod_elasticsearch2, no_automatic_wildcard, Context)) of
         true ->
             #{
                 <<"simple_query_string">> => #{
                     <<"query">> => Text,
                     <<"fields">> => Fields,
-                    <<"default_operator">> => <<"OR">>,
+                    <<"default_operator">> => DefaultOperator,
                     <<"flags">> => <<"ALL">>
                 }
             };
         false ->
             {SearchText, Ops} = add_wildcards(Text),
+            SearchText1 = iolist_to_binary([
+                    $(, SearchText, $),
+                    " | ",
+                    $", Text, $"
+                ]),
```

Comment:
I wonder if we should consider adding the exact-phrase-match as a separate (also configurable) thing? So it can also be applied to the no wildcards case.

Reply:
I think it is save to add it this way - especially as it an 'OR' with the loose-words.

Reply:
Ok, maybe something to revisit when confusion arises (if any) about what happens when wildcards are off, or when in some (unlikely) case where one would like to disable it.